### DISCUSSION TITLE
Npm ERR! code EBADPLATFORM

### DISCUSSION THREAD

**Discussion by pampapam2** [community/community#21523] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>Trying to install Juice Shop. First usage of Git.</p>
<p>What is the problem here:</p>
<p>.\juice-shop&gt;npm install<br>
npm ERR! code EBADPLATFORM<br>
npm ERR! notsup Unsupported platform for juice-shop@9.2.0: wanted {“os”:“win32,linux,darwin,freebsd,openbsd”,“arch”:“x64,x32,arm64,arm”} (current: {“os”:“win32”,“arch”:“ia32”})<br>
npm ERR! notsup Valid OS: win32,linux,darwin,freebsd,openbsd<br>
npm ERR! notsup Valid Arch: x64,x32,arm64,arm<br>
npm ERR! notsup Actual OS: win32<br>
npm ERR! notsup Actual Arch: ia32</p>
<p>System type: 32-bit operating system, x64-based processor</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/pampapam2">@pampapam2</a> </p>
<p>That’s not a git or github error, but the npm module you’re trying to install does not support windows in ia32 mode. You could either use a 64bit version of Windows or contact the developers of juice-show to ask for ia32 windows support.</p>
